packages/sirv/index.js

```diff
@@ -79,7 +79,11 @@ module.exports = function (dir, opts={}) {
 	if (opts.dev) {
 		return function (req, res, next) {
 			let stats, file, uri=decodeURIComponent(req.path || req.pathname || parser(req).pathname);
-			let arr = [uri].concat(toAssume(uri, extensions)).map(x => join(dir, x)).filter(fs.existsSync);
+			let arr = [uri]
+				.concat(toAssume(uri, extensions))
+				.map(x => normalize(join(dir, x)))
```

Comment:
What was the reason for `normalize` here?

Reply:
Sorry – you said in your PR.

Let's replace this with `resolve` since it does the same thing & `join`s at the same time:

```suggestion
				.map(x => resolve(dir, x))
```